index.js

```diff
@@ -0,0 +1,142 @@
+import {
+  startGame,
+  gameBord,
+  replayGame,
+  allcells,
+  nextPlayer,
+  playersForm,
+  playerOneName,
+  playerTwoName,
+  alertMsg,
+  gameOver,
+// eslint-disable-next-line import/extensions
+} from './dom.js';
+
+// eslint-disable-next-line func-names
+const gameModule = (function () {
+  const gameOn = false;
+  const playerOneName = '';
+  const playerTwoName = '';
+  const currentPlayerName = '';
+  const currentPlayerSym = '';
+  const gameBoard = ['', '', '', '', '', '', '', '', ''];
+  const count = 0;
+
+  return {
+    gameOn,
+    playerOneName,
+    playerTwoName,
+    currentPlayerName,
+    currentPlayerSym,
+    gameBoard,
+    count,
+  };
+}());
+
+const gameInit = (playerOneName, playerTwoName) => {
+  gameModule.gameOn = true;
+  gameModule.playerOneName = playerOneName;
+  gameModule.playerTwoName = playerTwoName;
+  gameModule.currentPlayerName = gameModule.playerOneName;
+  gameModule.currentPlayerSym = 'X';
+  playersForm.style.display = 'none';
+  nextPlayer.innerHTML = `${gameModule.currentPlayerName} Start!`;
+};
+
+const checkInput = (ev) => {
+  ev.preventDefault();
+  if (playerOneName.value === '' || playerTwoName.value === '') {
+    alertMsg.innerHTML = 'NAME CANNOT BE BLANK!';
+  } else {
+    gameInit(playerOneName.value, playerTwoName.value);
+  }
+};
```

Comment:
Let separate the DOM from application logic.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/